hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestBlockDataStreamOutput.java

```diff
@@ -70,6 +74,7 @@ public class TestBlockDataStreamOutput {
   private static String volumeName;
   private static String bucketName;
   private static String keyString;
+  private static final int DN_CURRENT_VERSION = DatanodeVersion.SEPARATE_RATIS_PORTS_AVAILABLE.toProtoValue();
```

Comment:
Should we use `DatanodeVersion#CURRENT_VERSION` here instead, or is this test trying to check that older datanodes return the correct version? I think the later is a good check, but we should probably not configure this old version in all the tests, as it may break things in the future. We should probably also change the variable name to `DN_OLD_VERSION` or something like that.

Reply:
The latter. Good point on the clarification part.